libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -695,4 +674,38 @@ class NavigationView : ConstraintLayout {
     companion object {
         private val TAG = NavigationView::class.java.simpleName
     }
+
+    init {
+        mapboxNavigationViewModel = ViewModelProvider(
+            viewModelStoreOwner
+        )[MapboxNavigationViewModel::class.java]
+        routeLineViewModel = ViewModelProvider(
+            viewModelStoreOwner,
+            RouteLineViewModelFactory(navigationViewOptions.mapboxRouteLineOptions)
+        )[RouteLineViewModel::class.java]
+        navigationStateViewModel = ViewModelProvider(
+            viewModelStoreOwner
+        )[NavigationStateViewModel::class.java]
+        cameraViewModel = ViewModelProvider(
+            viewModelStoreOwner
+        )[CameraViewModel::class.java]
+        lifecycle.addObserver(lifecycleObserver)
+        MapboxNavigationApp.setup(navigationOptions)
+            .attach(this)
+    }
+}
+
+private fun AttributeSet?.findAccessToken(context: Context): String? {
```

Comment:
Since you return `null` when `this == null`, you can just make receiver of the extension non-null and call it using `?.`. 
```suggestion
private fun AttributeSet.findAccessToken(context: Context): String? {
```